src/components/pages/RoleSignup/Applications/AppSuccess.js

```diff
@@ -1,28 +1,47 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Col, Row, Typography, Button } from 'antd';
+import { connect } from 'react-redux';
+import { getProfile } from '../../../../state/actions/userProfile/getProfile';
+import axiosWithAuth from '../../../../utils/axiosWithAuth';
 import { Link } from 'react-router-dom';
 const { Title } = Typography;
 
-const AppSuccess = () => {
+const AppSuccess = ({ isAuthenticated, getProfile }) => {
+  const [user, setUser] = useState({});
+  useEffect(() => {
+    axiosWithAuth()
+      .get(`/profile/current_user_profile/`)
+      .then(user => {
+        setUser(user.data);
+        getProfile(user.data.profile_id);
+      });
+  }, [isAuthenticated]);
```

Comment:
I think that an overarching code comment may prove useful here to quickly explain what this component is responsible for and the reasoning for the input of the dependency array _isAuthenticated_. What would be the result if this component were to re-render on the Authentication status change? Bear in mind I may be missing some insight into the entire workings of the codebase!